tests/autoinstall_test.go

```diff
@@ -156,6 +156,12 @@ var _ = Describe("kairos autoinstall test", Label("autoinstall-test"), func() {
 			Expect(out).To(ContainSubstring("/var/lib/longhorn"))
 		})
 
+		It("has rootfs shared mount", func() {
+			out, err := Sudo(`cat /proc/1/mountinfo | grep " / / "`)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
```suggestion
			Expect(err).ToNot(HaveOccurred(), out)
```
in case things break, this will provide more info

Reply:
good catch! thanks!